pypeit/processimages.py

```diff
@@ -536,6 +517,165 @@ def build_rawvarframe(self, trim=True):
         # Return
         return self.rawvarframe
 
+    # TODO Move these staticmethods to procimg
+    # This is a static method because I need to be able to run it from outside the class and would prefer
+    # to not have to create an instance of the class everytime I want to do that.
+    @staticmethod
+    def build_crmask(stack, proc_par, det, spectrograph, ivar=None, binning=None):
+        """
+        Generate the CR mask frame
+
+        Wrapper to procimg.lacosmic
+
+        Parameters
+        ----------
+        varframe : ndarray, optional
+
+        Returns
+        -------
+        self.crmask : ndarray
+          1. = Masked CR
+
+        """
+        # Run LA Cosmic to get the cosmic ray mask
+        varframe = utils.calc_ivar(ivar)
+        saturation = spectrograph.detector[det-1]['saturation']
+        nonlinear = spectrograph.detector[det-1]['nonlinear']
+        #sigclip, objlim = spectrograph.get_lacosmics_par(proc_par,binning=binning)
+        crmask = procimg.lacosmic(det, stack, saturation, nonlinear,
+                                  varframe=varframe, maxiter=proc_par['lamaxiter'],
+                                  grow=proc_par['grow'],
+                                  remove_compact_obj=proc_par['rmcompact'],
+                                  sigclip=proc_par['sigclip'],
+                                  sigfrac=proc_par['sigfrac'],
+                                  objlim=proc_par['objlim'])
+
+        # Return
+        return crmask
+
+    @classmethod
```

Comment:
I'm not keen on using class methods for this.
Would prefer to only use them for instantiation.

Please put this code in `core/procimg.py`

Reply:
See my comments below.